Subject: xli: makefile is not sysroot and generic compiler aware
From: Robert Schwebel <r.schwebel@pengutronix.de>

Use $(SYSROOT) in front of paths and $(CC) for the compiler.
Additionally, add some required libraries.

Signed-off-by: Robert Schwebel <r.schwebel@pengutronix.de>

---

Index: Makefile.std
===================================================================
--- Makefile.std.orig
+++ Makefile.std
@@ -7,17 +7,17 @@
 
 MISC_DEFINES=
 
-SYSPATHFILE=/usr/lib/X11/Xli
-INSTALLDIR=/usr/bin/X11
+SYSPATHFILE=$(SYSROOT)/usr/lib/X11/Xli
+INSTALLDIR=$(SYSROOT)/usr/bin/X11
 SHELL= /bin/sh
 MAKE= make
-STD_CC= cc
-GCC= gcc
+STD_CC= $(CC)
+GCC= $(CC)
 CP= cp
 LN= ln -s
 RM= rm -f
 MV= mv -f
-LIBS= -lX11 -lm
+LIBS= -lX11 -lm -ljpeg -lpng -lXext
 CFLAGS= -O -DSYSPATHFILE=\"$(SYSPATHFILE)\" $(OPTIONALFLAGS) $(EXTRAFLAGS)
 GCCFLAGS= -fstrength-reduce -finline-functions
 
